installation/resources/installer-cr-cluster.yaml.tpl

```diff
@@ -23,8 +23,14 @@ spec:
       namespace: "kyma-installer"
     - name: "istio-kyma-patch"
       namespace: "istio-system"
+    - name: "knative-serving-crds-init"
+      namespace: "knative-serving"
+    - name: "knative-serving-crds"
```

Comment:
If we really need a dedicated module with the CRDs, then let's please align it with a common way of doing it. We currently have velero-essentials, cluster-essentials and istio-init doinf similar things. Different naming and style, it is time to align it.

Reply:
I will create a followup ticket here.

Reply:
done.